src/components/MixedDrinks.spec.js

```diff
@@ -0,0 +1,32 @@
+import { render, screen } from '@testing-library/react'
+import MixedDrinks from './MixedDrinks'
+
+describe('MixedDrinks', () => {
+  const mixed = 'Shot'
+  const name = 'Wahnsinniger Bayleys'
+  const ingredience = 'Zutaten: Lakritzel und Bayleys'
+  const preparation =
+    'Zubereitung: Bayleys und Lakritzel zu gleichen Teilen in ein Pinnchen'
+
+  it('shows all Text', () => {
+    render(
+      <MixedDrinks
+        mixed={mixed}
+        name={name}
+        ingredience={ingredience}
+        preparation={preparation}
+      />
+    )
+
+    const text = screen.getByText('Shot - Wahnsinniger Bayleys')
```

Comment:
you can do:
`const text = screen.getByText(`${mixed] - ${name}`)`
Same for **text2** and **text3**

That said, Lene does not like it when you use variables in the tests.